pkg/app/piped/cloudprovider/kubernetes/diff_test.go

```diff
@@ -304,10 +316,83 @@ data:
 `,
 			diffNum: 1,
 		},
+		{
+			name: "Pod no diff 1",
+			manifests: `apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+      resources:
+        limits:
+          memory: "2Gi"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      resources:
+        limits:
+          memory: "2Gi"
+`,
+			expected:      "",
+			diffNum:       0,
+			falsePositive: false,
+		},
+		{
+			name: "Pod no diff 2",
+			manifests: `apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+      resources:
+        limits:
+          memory: "1.5Gi"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      resources:
+        limits:
+          memory: "1536Mi"
+`,
+			expected:      "",
+			diffNum:       0,
+			falsePositive: false,
+		},
 	}
 
 	for _, tc := range testcases {
+		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
Is this unnecessary?

Reply:
Thank you! By adding this, t.Run also runs concurrently. But this test is fast and so all t.parallel may not needed

Reply:
tbh, I think if we overdo on using routine, prepare for those routines may cause slower execution. So maybe, we can use parallel on test function level or test cases level only, wdyt? 🤔

Reply:
Oh, I got you. Pretty good!

Reply:
Agree about that. As the first step, parallel only at the high level (test function level) could be enough.

Reply:
We can use `-parallel` option in order to control that.
That's why we needn't to think about it every time.

Reply:
Thank you!
I'll just use t.parallel at the high level for now.